src/transformers/tokenization_utils.py

```diff
@@ -124,7 +177,7 @@ def __len__(self):
         """ Size of the full vocabulary with the added tokens """
         return self.vocab_size + len(self.added_tokens_encoder)
 
-    def add_tokens(self, new_tokens: Union[str, List[str]]) -> int:
+    def add_tokens(self, new_tokens: Union[str, List[str]], special_token=False) -> int:
```

Comment:
What's the rationale for adding this? It doesn't seem to be used below.

Reply:
It's used for the fast tokenizers here: https://github.com/huggingface/transformers/blob/master/src/transformers/tokenization_utils_fast.py#L209-L242

Reply:
Kept the same signature in both fast and slow tokenizers